pkg/manager/member/pd_member_manager.go

```diff
@@ -827,6 +827,15 @@ func clusterVersionGreaterThanOrEqualTo4(version string) (bool, error) {
 	return v.Major() >= 4, nil
 }
 
+func clusterVersionLessThan2(version string) (bool, error) {
```

Comment:
Why define this function here?

Reply:
Because dm-operator doesn't support dm-1.0 ...

Reply:
I mean why in this file?